src/mainwindow/mainwindow.h

```diff
@@ -33,6 +33,8 @@ class MainWindow : public QMainWindow
 
     void setupMainMenu();
 
+    void SaveState();
+    void RestoreState();
 private slots:
```

Comment:
Please change methods names according to our code convention (saveState(), restoreState())

Reply:
done